### PR TITLE
Implement Davis Distribution to Continuous Random Variables

### PR DESCRIPTION
**Pull request by jremes-foss**
#### Brief description of what is fixed or changed
This pull request adds the Davis distribution to the pre-built continuous random variables.

#### Release Notes

* stats
  * Adds the Davis distribution to the pre-built continuous random variables, incuding documentation and tests.

### CODE REVIEW

**Comment by hanspi42** [comment]
This looks fine, thank you. I will change the release notes a bit; once you agree, I will merge this PR.

**Comment by hanspi42** [comment]
Almost forgot: you have to doc/src/modules)/stats.rst, otherwise it will not appear in the documentation.

**Comment by jremes-foss** [comment]
@hanspi42 Thanks. Looks like I need to fix LaTeX in the documentation.

File: sympy/stats/crv_types.py [hunk 1]
```diff
@@ -1107,6 +1110,78 @@ def Dagum(name, p, a, b):
...
+def Davis(name, b, n, mu):
+    r""" Create a continuous random variable with Davis distribution.
+
+    Explanation
+    ===========
+
+    The density of Davis distribution is given by
+
+    .. math::
+        f(x; \mu; b, n) := \frac{b^{n}(x - \mu)^{1-n}}{ \left( e^{\frac{b}{x-\mu}} - 1 \right) \Gamma(n)\Zeta(n)}
```
Comment (hanspi42) [inline sympy/stats/crv_types.py:1143]: Small fix:
```
f(x; \mu; b, n) := \frac{b^{n}(x - \mu)^{1-n}}{ \left( e^{\frac{b}{x-\mu}} - 1 \right) \Gamma(n)\zeta(n)}
```

Comment (hanspi42) [reply sympy/stats/crv_types.py:1143]: The easiest way to chdck whether it works is to put it in a text cell of a Jupyter notbeook enclosed by `$$`

**Comment by hanspi42** [comment]
The documentation does not yet look OK:

https://output.circle-artifacts.com/output/job/43378255-d2b6-4521-8ba9-41cbdda4402c/artifacts/0/doc/_build/html/modules/stats.html#sympy.stats.Davis

**Comment by hanspi42** [comment]
This looks good; thank you for your contribution!

**Pull request merged**